srcts/src/bindings/input/date.ts

```diff
@@ -0,0 +1,312 @@
+import $ from "jquery";
+import { InputBinding } from "./InputBinding";
+import {
+  formatDateUTC,
+  updateLabel,
+  $escape,
+  parseDate,
+  hasOwnProperty,
+} from "../../utils";
+
+declare global {
+  interface JQuery {
+    // Adjustment of https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1626e0bac175121ec2e9f766a770e03a91843c31/types/bootstrap-datepicker/index.d.ts#L113-L114
+    bsDatepicker(methodName: string): any;
+    bsDatepicker(methodName: string, params: any): any;
+  }
+}
+
+type DateReceiveMessageData = {
+  label: string;
+  min?: Date;
+  max?: Date;
+  value?: Date;
```

Comment:
Also maybe empty array (i.e., `character(0)`)?
```suggestion
  min?: Date | null;
  max?: Date | null;
  value?: Date | null;
```